lms/djangoapps/ora_staff_grader/serializers.py

```diff
@@ -157,10 +159,55 @@ class Meta:
             "gradeStatus",
             "lockStatus",
             "score",
+            "email",
+            "fullname",
         ]
         read_only_fields = fields
 
 
+class AssessmentScoresMetadataSerializer(serializers.Serializer):
```

Comment:
I'd prefer: 
```suggestion
class AssessmentScoresSerializer(serializers.Serializer):
```
Since it's not quite metadata